### PR TITLE
[Multi Hub] Tab Stage View Unfurling: C#

### PR DESCRIPTION
**Pull request by Mohammed-MSFT**
Hi @ChetanSharma-msft @Prithvi-MSFT

Raied PR for Tab Stage View META-OS Unfurling Updates for  Csharp

Please review and provide feedback.

Thanks.

### CODE REVIEW

File: samples/tab-stage-view/csharp/README.md [hunk 1]
```diff
@@ -150,6 +158,18 @@ Please find below demo manifest which is deployed on Microsoft Azure and you can
 
 ![AppOutlook](TabInStageView/Images/AppOutlook.png)
 
+**After opening Outlook web, click the "New mail" button.**
+
+![Open New Mail](Images/OpenNewMail.png)
+
+**on the tool bar on top,select Apps icon. Your sideloaded app title appears among your installed apps**
```
Comment (ChetanSharma-msft) [inline samples/tab-stage-view/csharp/README.md:165]: **On the tool bar on top, select Apps icon. Your sideloaded app title appears among your installed apps**

File: samples/tab-stage-view/csharp/README.md [hunk 2]
```diff
@@ -150,6 +158,18 @@ Please find below demo manifest which is deployed on Microsoft Azure and you can
...
+
+![OpenAppIcon](Images/OpenAppIcon.png)
+
+**Opening stage view from unfurling link. If you copy and paste a link from https://www.BotFramework.com into the compose message area the link will unfurl.**
```
Comment (ChetanSharma-msft) [inline samples/tab-stage-view/csharp/README.md:169]: **Opening the stage view from unfurling link. If you copy and paste a link from https://www.BotFramework.com into the compose message area the link will unfurl.**

File: samples/tab-stage-view/csharp/README.md [hunk 3]
```diff
@@ -177,6 +197,3 @@ To learn more about deploying a bot to Azure, see [Deploy your bot to Azure](htt
 - [Tabs](https://learn.microsoft.com/microsoftteams/platform/tabs/what-are-tabs)
 - [Stage view](https://learn.microsoft.com/microsoftteams/platform/tabs/tabs-link-unfurling#stage-view)
 
-
-
-<img src="https://pnptelemetry.azurewebsites.net/microsoft-teams-samples/samples/tab-stage-view-csharp" />
```
Comment (ChetanSharma-msft) [inline samples/tab-stage-view/csharp/README.md:182]: Is this not required?

File: samples/tab-stage-view/csharp/TabInStageView/Bots/ActivityBot.cs [hunk 4]
```diff
@@ -69,6 +69,75 @@ protected override async Task OnMessageActivityAsync(ITurnContext<IMessageActivi
             await turnContext.SendActivityAsync(MessageFactory.Attachment(GetAdaptiveCardForStageView()));
         }
 
+        // <summary>
```
Comment (ChetanSharma-msft) [inline samples/tab-stage-view/csharp/TabInStageView/Bots/ActivityBot.cs:72]: /// <summary>

File: samples/tab-stage-view/csharp/TabInStageView/Bots/ActivityBot.cs [hunk 5]
```diff
@@ -69,6 +69,75 @@ protected override async Task OnMessageActivityAsync(ITurnContext<IMessageActivi
...
+        /// Overriding to invoke when an app based link query activity is received.
+        /// </summary>
+        protected override Task<MessagingExtensionResponse> OnTeamsAppBasedLinkQueryAsync(ITurnContext<IInvokeActivity> turnContext, AppBasedLinkQuery query, CancellationToken cancellationToken)
+        {
+            AdaptiveCard card = new AdaptiveCard(new AdaptiveSchemaVersion("1.2"))
```
Comment (ChetanSharma-msft) [inline samples/tab-stage-view/csharp/TabInStageView/Bots/ActivityBot.cs:77]: Can't we create json for this?
Also, 1.2 is the latest version?

File: samples/tab-stage-view/csharp/TabInStageView/Bots/ActivityBot.cs [hunk 6]
```diff
@@ -69,6 +69,75 @@ protected override async Task OnMessageActivityAsync(ITurnContext<IMessageActivi
...
+                    AttachmentLayout = "list",
+                    Type = "result",
+                    Attachments = new List<MessagingExtensionAttachment>
+                    {
+                        new MessagingExtensionAttachment
+                        {
+                             Content = card,
+                             ContentType = AdaptiveCard.ContentType,
+                             Preview = attachments,
+
```
Comment (ChetanSharma-msft) [inline samples/tab-stage-view/csharp/TabInStageView/Bots/ActivityBot.cs:134]: Remove extra empty lines.

File: samples/tab-stage-view/csharp/TabInStageView/Manifest/manifest.json [hunk 7]
```diff
@@ -50,9 +50,42 @@
     }
   ],
   "accentColor": "#60A18E",
+  "composeExtensions": [
+    {
+      "botId": "{{Microsoft-App-Id or Bot-id}}",
```
Comment (ChetanSharma-msft) [inline samples/tab-stage-view/csharp/TabInStageView/Manifest/manifest.json:55]: Not consistent placeholder name, check nodejs PR

File: samples/tab-stage-view/csharp/TabInStageView/Manifest_Hub/manifest.json [hunk 8]
```diff
@@ -39,9 +39,42 @@
     }
   ],
   "accentColor": "#60A18E",
+  "composeExtensions": [
+    {
+      "botId": "{{Microsoft-App-Id or Bot-id}}",
```
Comment (ChetanSharma-msft) [inline samples/tab-stage-view/csharp/TabInStageView/Manifest_Hub/manifest.json:44]: Not consistent placeholder name, check nodejs PR

**Review by ChetanSharma-msft — CHANGES_REQUESTED** [review]
Added my review comments.

**Comment by ChetanSharma-msft** [comment]
@Mohammed-MSFT - Kindly run build pipeline.

**Review by Prithvi-MSFT — APPROVED** [review]

**Comment by Prithvi-MSFT** [comment]
Build status - All builds passed
https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/5345357148

**Pull request merged**